doc/sphinx/source/introduction.rst

```diff
@@ -50,12 +50,13 @@ these challenging objectives the ESMValTool is conceived as a framework
 which allows community contributions to be bound into a coherent
 framework.
 
-.. _contact:
+.. _Support-and-Contact:
 
-Contact
+Support
```

Comment:
Here we should add the email contact to the User Engagement Team.

Reply:
May I have the email contact for the user engagement team?

Reply:
This is (without the space): esmvaltool_user_engagement_team @ listserv.dfn.de